titus-server-master/src/main/java/com/netflix/titus/master/eviction/service/quota/system/SystemQuotaController.java

```diff
@@ -94,6 +97,10 @@ private void resetDisruptionBudget(SystemDisruptionBudget newDisruptionBudget) {
         this.disruptionBudget = newDisruptionBudget;
         this.systemTokenBucket = newTokenBucket(newDisruptionBudget);
         this.inTimeWindowPredicate = TimeWindowFunctions.isInTimeWindowPredicate(titusRuntime, disruptionBudget.getTimeWindows());
+        this.quotaMessage = String.format("System quota token bucket: capacity=%s, refillRate=%s",
```

Comment:
Is refillRate an actual rate or the name of the strategy?

Reply:
Renamed to `refillStrategy`